v4/source/definitions.yaml

```diff
@@ -116,6 +116,10 @@ components:
       properties:
         total_users_count:
           type: integer
+    KnownUsers:
+        type: array
+          items:
```

Comment:
I think there is just an indentation issue here

Reply:
Thank you @BenCookie95  Fixed that!